resources/web/docs_js/components/console_widget.jsx

```diff
@@ -33,23 +33,25 @@ export class _ConsoleForm extends Component {
 
     return (
       <form>
+         <p><strong>Want to use the Kibana Console?</strong></p>
         <label for="url">{props.langStrings(props.url_label)}</label>
         <input
           id="url"
           type="text"
           value={getValueFromState('url')}
           onInput={linkState(this, getFieldName('url'))}
         />
-
-        <label for="curl_host">curl {props.langStrings('host')}</label>
+        <p></p>
```

Comment:
Below `Console URL` field:

Go to **Dev Tools > Console**, then copy the URL.

Reply:
I've cut things way back in my latest commit

Reply:
Going with "Learn more about the Dev Tools [Console](https://www.elastic.co/guide/en/kibana/current/console-kibana.html)."